### PR TITLE
Code quality recommendations.

### PR DESCRIPTION
**Pull request by mike1729**

### CODE REVIEW

File: src/extender.rs [hunk 1]
```diff
@@ -269,21 +262,26 @@ impl<H: Hasher> Extender<H> {
...
-                self.finalize_round(self.state.current_round, &candidate_hash)?;
-                self.state.current_round += 1;
-                self.state.round_initialized = false;
-                continue;
-            }
-            if decision == Some(false) {
-                self.state.pending_cand_id += 1;
-                self.state.votes_up_to_date = false;
-                continue;
+            match decision {
```
[the remark this answers is not in the document]
Comment (timorl) [reply src/extender.rs:265]: Although I guess leaving the `break` might actually be more readable. Anyway, `continue`s can go.

File: src/network.rs [hunk 2]
```diff
@@ -123,12 +123,12 @@ impl<H: Hasher, D: Data, S: Signature, MS: PartialMultisignature, N: Network<H,
         match recipient {
             Everyone => {
                 if let Err(error) = self.network.broadcast(data) {
-                    error!(target: "AlephBFT-etwork-hub", "Broadcast error: {:?}", error);
+                    error!(target: "AlephBFT-network-hub", "Broadcast error: {:?}", error);
```
Comment (timorl) [inline src/network.rs:126]: Oh no, this will be a conflict with the logging PR where I also do it...

**Review by timorl — CHANGES_REQUESTED** [review]

File: src/extender.rs (hunk 1 above)
[the remark this answers is not in the document]
Comment (mike1729) [reply src/extender.rs:265]: `continue`s are gone.

File: src/network.rs (hunk 2 above)
Comment (mike1729) [reply src/network.rs:126]: :(

**Review by timorl — APPROVED** [review]
<3

**Pull request merged**